src/models/api/DeploymentUpdateRequest.ts

```diff
@@ -13,4 +13,5 @@ export type DeploymentUpdateRequest = Partial<{
   infrastructure_document_id: string | null,
   work_queue_name: string | null,
   work_pool_name: string | null,
+  infra_overrides: unknown | null,
```

Comment:
For the request there's definitely a type the API is expecting, I don't think `unknown` is appropriate here